packages/core/src/IconButton.js

```diff
@@ -19,7 +19,15 @@ const TransparentButton = styled(Button)`
 const IconButton = ({ name, size, color, ...props }) => (
   <TransparentButton {...props}>
     <div>
-      <Icon name={name} size={size} color={color} />
+      <Icon
+        name={name}
+        size={size}
+        color={color}
+        title={props.title}
```

Comment:
any reason not to just spread the remaining props `{...props}`?

Reply:
It caused problems when I attached an `onClick()` handler. You end up with a ghost click due to the same handler being passed to `Icon` as well as `IconButton`.

Reply:
for a11y reasons, there should be a button wrapped around the icon instead of a direct click handler on the button, but manually mapping the props is fine.

Reply:
> It caused problems when I attached an onClick() handler. You end up with a ghost click due to the same handler being passed to Icon as well as IconButton.

Interesting observation. @bryantp, can you provide an example (would be great if a CodeSandbox link) to understand more about why `...props` would fail this use case.